### ISSUE TITLE
Upload become slow on 37.0.5

### ISSUE THREAD

**Issue by eukaryoteoops** [robinrodricks/FluentFTP#883]
**FTP OS:** Unix

**FTP Server:** ProFTPD

**Computer OS:** Windows

**FluentFTP Version:** 37.0.5

<write details about your bug report / feature request here>

**Logs :**
```

# OpenPassiveDataStreamAsync(PASV, "STOR 11111/11111s-2.jpg", 0)
Command:  PASV
Response: 227 Entering Passive Mode (10,140,0,120,27,168).
Status:   Connecting to 34.80.150.113:7080
Command:  STOR 11111/11111s-2.jpg
Response: 150 Opening BINARY mode data connection for 11111/11111s-2.jpg
Status:   Disposing FtpSocketStream...

```
**MyCode:**
```
await _ftp.AutoConnectAsync();
foreach (var item in imageMapping)
       await _ftp.UploadAsync(item.Value, $"{cid}/{item.Key}", FtpRemoteExists.Overwrite, true);
 await _ftp.DisconnectAsync();`
```

I try to use older version (37.0.0), it would not pending on `Disposing FtpSocketStream...`, 10 files takes 2 seconds,
but 37.0.5, 10 files takes 2 minutes to finish uploading, logs shows each file pending on `Disposing FtpSocketStream...`

please help to resolve,thanks

**Comment by robinrodricks** [comment]
Thanks very much for this report.

**Comment by robinrodricks** [comment]
I think it was caused by this recent change - https://github.com/robinrodricks/FluentFTP/pull/854

I will revert it and send you a build to check.

**Comment by robinrodricks** [comment]
Hi can you check if this resolves your issue.
[Release.zip](https://github.com/robinrodricks/FluentFTP/files/9082611/Release.zip)

**Comment by robinrodricks** [comment]
Please use:

https://www.nuget.org/packages/FluentFTP/37.0.6

**Comment by eukaryoteoops** [comment]
it works on 37.0.6 as older version, thanks for your help~

**Comment by robinrodricks** [comment]
Awesome!

**Comment by Noxxys** [comment]
@robinrodricks does this mean that we won't have the fix for the TLS socket incorrectly closed anymore?

**Comment by robinrodricks** [comment]
@Noxxys if you can replicate the issue that was fixed by that PR, please raise a new ticket.
